.gitlab-ci.yml

```diff
@@ -13,9 +13,11 @@
 # limitations under the License.
 
 
-
+variables:
+  PATH: "/home/gitlabci/bin:/usr/lib64/qt-3.3/bin:/home/gitlabci/bin:/home/gitlabci/bin:/bin:/usr/bin:/usr/local/bin:/sbin:/usr/sbin:/usr/local/sbin:/usr/local/condor/bin:/usr/sepp/bin:/usr/sepp/bin:/usr/local/anaconda3/bin:/var/lib/snapd/snap/bin:/usr/sepp/bin:/usr/local/anaconda3/bin:/var/lib/snapd/snap/bin:/usr/local/condor/bin:/usr/sepp/bin:/usr/sepp/bin:/var/lib/snapd/snap/bin"
```

Comment:
You should deduplicate entries in this `PATH`. For every entry, you remove all occurrences but the first, then you have a minimal `PATH` that is equivalent to the current one.

Reply:
Done! :)